squish.ini

```diff
@@ -62,4 +62,6 @@ attachableAUT/AUT_150418 = "localhost:61500"
 attachableAUT/AUT_150532 = "localhost:61500"
 attachableAUT/AUT_150639 = "localhost:61500"
 attachableAUT/AUT_150925 = "localhost:61500"
+attachableAUT/AUT_171350 = "localhost:61500"
+attachableAUT/AUT_171701 = "localhost:61500"
 
```

Comment:
I don't think we need to commit this file @Valentina1133

Reply:
Ooops! Will remove it, thank you 🙏🏻

Reply:
Done